public/content/en/gems/ndslice.md

```diff
@@ -0,0 +1,120 @@
+# Multidimensional arrays
+
+D provides support for multidimensional arrays directly in its standard library
+Phobos. It allows fast iteration, access and manipulation of multidimensional
+data. In comparison to popular software packages in other languages (e.g. `ndarray`
+in NumPy or `Array` in Julia), it doesn't become slow for computations not
+covered by preexisting functionality in a bound C-library nor forces the user
+to escape into another language. Additionally, due to its compile-time introspection
+capabilities, D enables further optimizations for superior performance.
+
+## Slices
+
+A `Slice` is a view on a multidimensional view on usually an underlying raw,
+one-dimensional array. With the `sliced` function a multidimensional array can
+be constructed - in this example a `2x3` matrix is created:
```

Comment:
The `sliced` function is the simplest entrypoint into the world ndslice. In the following example it used to create a 2x3 matrix view over the existing static array of length 6.

[Some ascii art diagram]
```
arr:
arr[0] arr[1] arr[2] arr[3] arr[4] arr[5]

matrix:
                      col0                    col1                      col2
row0: m[0, 0] == arr[0]  m[0, 1] == arr[1]   m[0, 2] == arr[2]
row1: m[1, 0] == arr[3]  m[1, 1] == arr[4]   m[1, 2] == arr[5]
```